docs/source/user_guide.rst

```diff
@@ -114,17 +114,33 @@ Using the functions:
 
     .. code-block:: python
 
-        from einsteinpy import utils
+        import numpy as np
+        from astropy import units as u
+f       from einsteinpy.coordinates import BoyerLindquistDifferential, CartesianDifferential, Cartesian, BoyerLindquist
+
+        a = 0.5 * u.km
+
+        pos_vec = Cartesian(.265003774 * u.km, -153.000000e-03 * u.km,  0 * u.km)
+
+        bl_pos = pos_vec.to_bl(a)
+        print(bl_pos)
+
+        cartsn_pos = bl_pos.to_cartesian(a)
+        print(cartsn_pos)
+
+        pos_vel_coord = CartesianDifferential(.265003774 * u.km, -153.000000e-03 * u.km,  0 * u.km,
+                                  145.45557 * u.km/u.s, 251.93643748389 * u.km/u.s, 0 * u.km/u.s)
+
+        bl_coord = pos_vel_coord.bl_differential(a)
+        bl_coord = np.array(bl_coord)
```

Comment:
I don't think this line would work.
I guess what to you want is `bl_coord = bl_coord.si_values(); bl_vel = bl_coord[3:]`
Try them on your system before changing it here :)

Reply:
I tried both np.array and si_values
Both worked on my system

Reply:
are you sure? lemme check!

Reply:
gimme a screenshot. No method for this is present. How is this working??

Reply:
I tried it on a rough jupyter notebook.
And I made the changes you suggested . I thought maybe someone may not want to change the units that why I chose np.array 



<img width="937" alt="Screen Shot 2019-06-25 at 2 56 59 PM" src="https://user-images.githubusercontent.com/26502870/60086559-e8ca5080-9758-11e9-9d43-c9efccd20a4a.png">

Reply:
As I can see there is no difference b/w `bl_coord` and `np.array(bl_coord)`

Reply:
you are right! I misunderstood

Reply:
I changed it to si_values()